cmd/kube-apiserver/app/options/options.go

```diff
@@ -162,6 +164,9 @@ func (s *ServerRunOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.SSHKeyfile, "ssh-keyfile", s.SSHKeyfile,
 		"If non-empty, use secure SSH proxy to the nodes, using this user keyfile")
 
+	fs.StringSliceVar(&s.SSHPreferredAddressTypes, "ssh-preferred-address-types", s.SSHPreferredAddressTypes,
+		"List of the preferred NodeAddressTypes to use for SSH Proxy.")
```

Comment:
which are allowed?

Reply:
Where is the validation?

Reply:
"SSH proxy"

Reply:
- Added the list of allowed address types in f6d8533282957abc99155e89f07ad3d861a4aa80
- Added validation in 4f0a6c6b30aac7c838547eae713b06333a15915c